common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -32,6 +32,15 @@
 from embargo.test_utils import restrict_course
 
 
+class UrlMixin(object):
+    """
+    Mixins for URL processing
+    """
+    def get_absolute_url(self, path):
+        """ Generate an absolute URL for a resource on the test server. """
+        return u'http://testserver/{}'.format(path.lstrip('/'))
+
+
```

Comment:
@Nickersoft a few issues with this:

* if `http://testserver/` is the url prefix in order to match something defined in settings, then please substitute the setting value in the format string instead of hardcoding it here.
* this is a fairly trivial piece of logic and I don't think it merits the added code / complexity of using a mixin.  
* Could you just use [`HttpRequest.build_absolute_uri`](https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.build_absolute_uri) and skip extra code entirely?